bazel/envoy_build_system.bzl

```diff
@@ -65,6 +53,26 @@ def envoy_linkopts():
                                    ["-static-libstdc++", "-static-libgcc"]) \
     + envoy_select_exported_symbols(["-Wl,-E"])
 
+def _envoy_stamped_linkopts():
+  return select({
+    "@bazel_tools//tools/osx:darwin": [
+        "-sectcreate,__TEXT,__build_id", "$(location //bazel:raw_build_id.ldscript)"
+    ],
+    "//conditions:default": [
+        "-Wl,@$(location //bazel:gnu_build_id.ldscript)",
```

Comment:
Wondering if this needs to be `@envoy//bazel...` or similar in the envoy-filter-example scenario.

Reply:
I successfully built `envoy-filter-example` pointing to an Envoy src dir with my changes, and it has the correct build ID.